invokeai/frontend/web/public/locales/en.json

```diff
@@ -1044,7 +1044,7 @@
         "parameterNotSet": "{{parameter}} not set",
         "parameterSet": "{{parameter}} set",
         "parametersNotSet": "Parameters Not Set",
-        "parametersSet": "Parameters Set",
+        "parametersSet": "Parameters",
```

Comment:
This is going to confuse translators, let's rename this to just `parameters` if it doesn't need " Set" in the localized string